integration-cli/docker_cli_events_test.go

```diff
@@ -686,3 +686,82 @@ func (s *DockerRegistrySuite) TestEventsImageFilterPush(c *check.C) {
 		c.Fatalf("Missing 'push' log event for image %s\n%s", repoName, out)
 	}
 }
+
+func (s *DockerSuite) TestEventsOOMDisableFalse(c *check.C) {
```

Comment:
this tests is failing with wrong events number :/ (run locally, also `dockerCmdWithError` doesn't accept `c` as first parameter at line 693)